Topl/BramblSc
# TSDK-766: Monitoring Tests

## Purpose

Add tests for the work done in https://github.com/Topl/BramblSc/pull/177

## Approach

- Added tests for the Bitcoin Block Monitor; 2 tests, one to only report new blocks, another to report new + existing blocks
- Added the tests to the CICD workflow

## Testing

- Ensure it ran as expected locally (using the same docker image)
- Verify workflow run is successful https://github.com/Topl/BramblSc/actions/runs/8348317478
- Publish test result now has +2 more tests https://github.com/Topl/BramblSc/pull/178#issuecomment-2007829204

## Tickets
* Closes TSDK-766
